src/design-system/components/checkbox/CheckBox.tsx

```diff
@@ -12,9 +12,16 @@ interface CheckBoxProps extends Omit<CheckboxProps, 'size'> {
   color?: 'primary' | 'secondary' | 'error' | 'success'
   size?: 'sm' | 'md' | 'lg'
   loading?: boolean
+  checked?: boolean
+  onChange?: (
+    event: React.ChangeEvent<HTMLInputElement>,
+    checked: boolean
+  ) => void
 }
 
 const CheckBox: FC<CheckBoxProps> = ({
+  checked,
```

Comment:
`checked` is a boolean, so rename it to `isChecked`. But also, to be more consistent in naming in this component, you should use rename while destructuring.
So you will have:
```isChecked: externalIsChecked```